### PR TITLE
so that I can communicate to all district staff as a RTS staff role, I can send sms

### PR DESCRIPTION
**Pull request by gitumarkk**
The purpose of this is so that the rts staff who don't have the district admin role can send an sms.

### CODE REVIEW

**Comment by gitumarkk** [comment]
@imsickofmaps ready for review.

File: sms/admin.py [hunk 1]
```diff
@@ -162,6 +169,97 @@ def save_related(self, request, form, formsets, change):
...
+            return HttpResponseRedirect(reverse("admin:index"))
+        # Replicating views from django ModelAdmin
+        model = self.model
+        opts = model._meta
+
+        # getting all the districts qs
+        districts = District.objects.all()
+
+        if request.method == "POST":
+            district_form = ChooseDistricsForm(request.POST, queryset=districts)
```
Comment (imsickofmaps) [inline sms/admin.py]: Typo `ChooseDistricsForm`

File: sms/admin.py [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+from rts.actions import export_select_fields_csv_action
 from sms.models import SendSMS, SMSZones, TempSMSZones
-from users.models import UserDistrict
 from hierarchy.models import Zone, District
-from django import forms
-from django.forms.models import BaseInlineFormSet, inlineformset_factory
-from django.contrib.admin.util import flatten_fieldsets
-from functools import partial
-from django.db import models
+from sms.forms import ChooseDistricsForm
```
Comment (imsickofmaps) [inline sms/admin.py]: Typo `ChooseDistricsForm`

File: sms/forms.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from django.utils.translation import ugettext_lazy as _
+from django import forms
+
+
+class ChooseDistricsForm(forms.Form):
```
Comment (imsickofmaps) [inline sms/forms.py]: Typo `ChooseDistricsForm`

File: sms/forms.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    send_to_all = forms.BooleanField(label=_("All"), required=False)
+
+    def __init__(self, *args, **kwargs):
+        queryset = kwargs.pop("queryset", None)
+        super(ChooseDistricsForm, self).__init__(*args, **kwargs)
+        for query in queryset:
+            self.fields["%s" % query.id ] = forms.BooleanField(label=query.name, required=False)
+
+    def clean(self):
+        cleaned_data = super(ChooseDistricsForm, self).clean()
```
Comment (imsickofmaps) [inline sms/forms.py]: Typo `ChooseDistricsForm`

**Comment by imsickofmaps** [comment]
@gitumarkk needs the typo fixing throughout

**Comment by gitumarkk** [comment]
@imsickofmaps ready for re-review.

**Comment by imsickofmaps** [comment]
@smn also ready now

**Comment by imsickofmaps** [comment]
Buggy branch. Merged in PR #89

**Pull request closed without merging**